pkg/kernel/tools/nshandle/ns_handle.go

```diff
@@ -77,3 +100,18 @@ func RunIn(current, target netns.NsHandle, runner func() error) error {
 
 	return runner()
 }
+
+// SetLinkUp makes the interface to operationally up
+func SetLinkUp(ifName string) error {
```

Comment:
Doesn't look like this method should be located under `nshandle`. Also, all usages of this method are like:
```go
err = nshandle.SetLinkUp(ifName)
if err != nil {
	return errors.WithStack(err)
}

l, err := handle.LinkByName(ifName)
if err != nil {
	return errors.WithStack(err)
}
```
So probably instead of this it would be better to just inline it in all places and remove from here?

Reply:
> So probably instead of this it would be better to just inline it in all places and remove from here?

@Bolodya1997 not sure i fully understand this, can you check the method now ? is that what you're looking for ?

Reply:
yes! done.